src/script/components/calling/CallingCell.tsx

```diff
@@ -251,10 +259,10 @@ const CallingCell: React.FC<CallingCellProps> = ({
 
   useEffect(() => {
     if (isIncoming) {
-      window.addEventListener('keydown', answerOrRejectCall);
+      document.addEventListener('keydown', answerOrRejectCall, {capture: true});
```

Comment:
Can you just add a comment to why you add a `capture` here, would be nice :)

Reply:
Since this is rarely used, I think it would be nice to have a trace of the explanation in the code

Reply:
I'd appreciate it :)